pytorch_lightning/metrics/metric.py

```diff
@@ -145,6 +145,7 @@ def add_state(
         self._defaults[name] = deepcopy(default)
         self._reductions[name] = dist_reduce_fx
 
+    @torch.jit.unused
```

Comment:
`@torch.jit.unused` and `@torch.jit.ignore` both work fine. 
I tested them with https://colab.research.google.com/drive/1MscNHxIc_LIbZxALHbZOAkooNu0TzVly?usp=sharing